src/FinishBookForm/FinishBookForm.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+export const FinishBookForm = () => {
+    const [rating, setRating] = useState(0);
+    const [review, setReview] = useState('');
+
+    return(
+        <form>
+          <label>
+          How did you like it on a scale from 1 to 5?
+            <input
+              aria-label="rating input"
+              className="rating-input"
+              placeholder="choose a number between 1 and 5"
```

Comment:
we probably want the rating to be a row of clickable stars/apples instead of a text input. This version is just a placeholder that will get us the data we want to POST

Reply:
Agreed!